src/css/tailwind.css

```diff
@@ -411,6 +413,41 @@ main[class*='docMainContainer'] .col--3,
   margin: 0 auto;
 }
 
+.docItemContainer_node_modules-\@docusaurus-theme-classic-lib-theme-DocItem-Layout-styles-module header + *, .docItemContainer_node_modules-\@docusaurus-theme-classic-lib-theme-DocItem-Layout-styles-module article > *:first-child {
```

Comment:
This selector will not work on the built project because it removes everything after docItemContainer and adds a different generated string.

![image](https://user-images.githubusercontent.com/36553513/195335537-37918e47-3d57-4c4d-b10d-28d124c5336d.png)